src/handlers/handlers.rs

```diff
@@ -42,6 +46,21 @@ pub async fn get_stats_from_linux(sys: PlatformImpl) -> Stats {
             println!("\nMeasuring CPU load...");
             thread::sleep(Duration::from_secs(1));
             let cpu = cpu.done().unwrap();
+
+            let load_avg = match sys.load_average() {
+                Ok(load) => Loadavg {
+                    one: load.one,
+                    five: load.five,
+                    fifteen: load.fifteen,
+                },
+                Err(err) => panic!("Error: {:?}", err),
```

Comment:
I think panic just halts the program afaik, we need to both println(for now, later we'll use application logs to log it in a file or to an output buffer), and maybe use a default value, so that memory_details is assigned that default value.